docs/source/index.rst

```diff
@@ -76,7 +76,7 @@ There are many ways to get involved in the development of cartopy:
 
  * If you write a paper which makes use of cartopy, please consider :doc:`citing <citation>` it.
  * If you publish the maps and plots, please consider the required :ref:`attribution of copyright  <referencing_copyright>` for the data. 
- * Report bugs to https://github.com/SciTools/cartopy/issues (please look to see if there are any outstanding
+ * Report bugs and problems with the code or documentation to https://github.com/SciTools/cartopy/issues (please look to see if there are any outstanding
```

Comment:
Many of the tickets here are not bug reports, but other things like feature requests, clarifications, discussions about potential extensions and optimizations. That kind of thing.